beps/bep_0011.rst

```diff
@@ -87,9 +87,37 @@ An added contact must not be dropped in the same message.
 
 Except for the initial PEX message the combined amount of added v4/v6 contacts should not exceed 50 entries. The same applies to dropped entries.
 
+    **Implementation Note**: If these limits conflict with large connection pools then implementations can calculate the canonical peer priorities from the perspective of a remote peer to choose which subset of
+connections to prioritize for inclusion in a message.  
```

Comment:
It might be a good idea to reference BEP 40 here

Reply:
It's already referenced somewhere in the BEP, but I can add it here too.

Reply:
Actually, I'll remove that. It may have unintended consequences such as biasing the result set too much. So it may not be good advice anyway.